.github/workflows/rebase.yml

```diff
@@ -235,6 +237,7 @@ jobs:
           ${{ github.workspace }}/.github/scripts/rebase-finish.sh ${{ env.REBASE_BRANCH }} ${{ env.ORIGIN_BRANCH }}
 
   deploy:
+    if: github.ref == 'refs/heads/woarm64'
```

Comment:
Why is this check needed?

Reply:
Deployment is not allowed from other branch than \`woarm64\`.